### DISCUSSION TITLE
Some plugins are missing in Ubuntu packages for releases past Bionic

### DISCUSSION THREAD

**Discussion by starteleport** [strongswan/strongswan#1256] [Q&A]
Hi!

Actually I've posted a copy of this on https://bugs.launchpad.net/ubuntu/+source/strongswan/+bug/1987909, but it does not seems to be too popular.

I'm installing `libcharon-extra-plugins` on Ubuntu Focal and expecting `attr-sql` and friends to be there. However these plugins are only included for Bionic release. Is this intentional?

**Comment by Thermi** [comment]
Not an issue that we can help here. This is an Ubuntu packaging issue, not a strongSwan one.
